components/aws/sagemaker/tests/integration_tests/component_tests/test_groundtruth_component.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+import os
+import json
+import utils
+from utils import kfp_client_utils
+from utils import sagemaker_utils
+from test_workteam_component import create_workteamjob
+import time
+
+
+@pytest.mark.parametrize("test_file_dir", ["resources/config/image-classification-groundtruth"])
```

Comment:
There were markers placed on some of the tests for canaries. Would you mind adding these for your components, too?

Reply:
Are we running all integ tests as canaries?

Reply:
It seems like we are so far. Maybe once we start doing feature-specific integration tests we will change that.

Reply:
Done